packages/web-components/src/components/textarea/textarea.stories.ts

```diff
@@ -200,19 +202,93 @@ export const Skeleton = {
 };
 
 export const WithAILabel = {
-  render: () => html`
-    <cds-textarea label="Text Area label" helper-text="Optional helper text">
+  args,
+  argTypes,
+  render: ({
+    cols,
+    counterMode,
+    disabled,
+    enableCounter,
+    helperText,
+    hideLabel,
+    invalid,
+    invalidText,
+    label,
+    maxCount,
+    onInput,
+    placeholder,
+    readonly,
+    rows,
+    value,
+    warn,
+    warnText,
+  }) => html`
+    <cds-textarea
+      ?enable-counter="${enableCounter}"
+      counter-mode="${ifDefined(counterMode)}"
+      helper-text="${ifDefined(helperText)}"
+      ?hide-label="${hideLabel}"
+      ?invalid="${invalid}"
+      invalid-text="${ifDefined(invalidText)}"
+      label="${ifDefined(label)}"
+      ?readonly="${readonly}"
+      value="${ifDefined(value)}"
+      ?warn="${warn}"
+      warn-text="${ifDefined(warnText)}"
+      ?disabled="${disabled}"
+      max-count="${ifDefined(maxCount)}"
+      placeholder="${ifDefined(placeholder)}"
+      @input="${onInput}"
+      rows="${ifDefined(rows)}"
+      cols="${ifDefined(cols)}">
       <cds-ai-label alignment="bottom-right">
-        ${content}${actions}</cds-ai-label
-      >
+        ${content}${actions}
+      </cds-ai-label>
     </cds-textarea>
   `,
 };
 
 export const WithLayer = {
-  render: () => html`
+  args,
```

Comment:
```suggestion
  args: { ...args, helperText: 'Optional helper text' },
```

Same here, lets sync with React story.